pkg/jwtmanager/jwtmanager.go

```diff
@@ -21,8 +21,10 @@ import (
 
 // VouchClaims jwt Claims specific to vouch
 type VouchClaims struct {
-	Username string   `json:"username"`
-	Sites    []string `json:"sites"` // tempting to make this a map but the array is fewer characters in the jwt
+	Username    string   `json:"username"`
```

Comment:
I've been wondering if this should be `sub` instead of `username`.  Do you have an opinion?

Reply:
No opinion from my side! :)